libraries/tlv-account-resolution/src/state.rs

```diff
@@ -158,65 +172,92 @@ impl ExtraAccountMetas {
         }
     }
 
-    /// Add the additional account metas to an existing instruction
-    pub fn add_to_vec<T: SplDiscriminate>(
-        account_metas: &mut Vec<AccountMeta>,
-        data: &[u8],
-    ) -> Result<(), ProgramError> {
-        let state = TlvStateBorrowed::unpack(data)?;
-        let bytes = state.get_first_bytes::<T>()?;
-        let extra_account_metas = PodSlice::<PodAccountMeta>::unpack(bytes)?;
-        let initial_instruction_length = account_metas.len();
-        for mut account_meta in extra_account_metas.data().iter().map(AccountMeta::from) {
-            Self::de_escalate_account_meta(
-                &mut account_meta,
-                account_metas,
-                initial_instruction_length,
-            );
-            account_metas.push(account_meta);
+    /// Resolve a program-derived address (PDA) from the instruction data
+    /// and the accounts that have already been resolved
+    fn resolve_pda(
+        resolved_metas: &[AccountMeta],
+        instruction_data: &[u8],
+        program_id: &Pubkey,
+        seeds: &[Seed],
+    ) -> Result<Pubkey, ProgramError> {
+        let mut pda_seeds: Vec<&[u8]> = vec![];
+        for config in seeds {
+            match config {
+                Seed::Uninitialized => continue,
```

Comment:
nitty nit take-it-or-leave-it: rather than a `continue`, can you just do `{}`? it doesn't make a difference here, but typically `break` and `continue` make flow control much harder to reason.

Reply:
`()` ?

Reply:
Whichever you prefer!